mix.exs

```diff
@@ -51,9 +51,11 @@ defmodule Membrane.Audiometer.Plugin.Mixfile do
 
```

Comment:
`mix.exs` housekeeping:
* nesting prefix in docs
* formatters in docs
* Add license to extras (without it, IIRC `mix docs` will generate a warning of missing file reference)
* add `-` in license
* bump minimal Elixir version (core requires 1.12)
* compare to the template and check if I haven't missed anything